src/PCA9633.cpp

```diff
@@ -133,6 +133,20 @@ void PCA9633::setLdrStateAll(uint8_t state) {
     writeReg(REG_LEDOUT, newReg);
 }
 
+void PCA9633::setDrvState(uint8_t state) {
+
+    uint8_t prevReg = readReg(REG_MODE2);
+    uint8_t newReg;
+
+    // first clear both bits of drv
+    newReg = prevReg & ~(0b11 << BIT_OUTDRV);
```

Comment:
Clearing two bits would also have an effect on `OCH` in the `MODE2` register. Only the outdrv bit should be changed.